src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/FeedbackStreamingEventArgs.cs

```diff
@@ -0,0 +1,52 @@
+﻿/*
+* Copyright (c) 2019 Samsung Electronics Co., Ltd All Rights Reserved
+*
+* Licensed under the Apache License, Version 2.0 (the License);
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an AS IS BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+
+namespace Tizen.Uix.VoiceControlManager
+{
+    /// <summary>
+    /// This Class contains the pre recognition results(partial ASR) from vc-daemon.
+    /// </summary>
+    public class FeedbackStreamingEventArgs : EventArgs
+    {
+        internal FeedbackStreamingEventArgs(FeedbackEventType evt, string buffer)
+        {
+            FeedbackEvent = evt;
+            Buffer = buffer;
+        }
+
+        /// <summary>
+        /// TTS feedback event
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public FeedbackEventType FeedbackEvent
+        {
+            get;
+            private set;
+        }
+
+        /// <summary>
+        /// Audio streaming data
```

Comment:
Is it right description?
How to represent a audio data in string?

Reply:
Fixed.

Reply:
Could you explain, how to represent a audio data in string?

Reply:
Changed string type to byte[].